cellprofiler/modules/measureradialentropy.py

```diff
@@ -0,0 +1,205 @@
+'''<b>MeasureRadialEntropy</b> measures the variability of an image's
+intensity inside a certain object
+<hr>
+<p>MeasureRadialEntropy divides an object into pie-shaped wedges and
+ measures either the mean or median intensity of each.  Once the intensity
+ of each wedge has been calculated, the entropy of the bin totals is calculated.</p>
+
+ <p>This module is under construction</p>
+
+'''
+
+
+import numpy as np
+from scipy import stats
+
+
+
+import cellprofiler.module as cpm
+import cellprofiler.measurement as cpmeas
+import cellprofiler.object as cpo
+import cellprofiler.setting as cps
+
+from centrosome.cpmorphology import minimum_enclosing_circle
+
+
+
+ENTROPY = "Entropy"
```

Comment:
Global variables are [considered](http://programmers.stackexchange.com/questions/148108/why-is-global-state-so-evil) a [code smell](https://en.wikipedia.org/wiki/Code_smell) and should be avoided. While there’re a number of reasons for this, the two most common critiques are they make it difficult to refactor using automated tools (e.g. PyCharm) and they introduce state inside module scope that’s difficult to reason about.